runtime/src/main/java/org/teiid/transport/PGCharsetConverter.java

```diff
@@ -53,6 +53,7 @@
         mapCharset("SJIS", Charset.forName("windows-932")); //$NON-NLS-1$ //$NON-NLS-2$
         mapCharset("UHC", Charset.forName("windows-949")); //$NON-NLS-1$ //$NON-NLS-2$
         mapCharset("UTF8", Charset.forName("UTF-8")); //$NON-NLS-1$ //$NON-NLS-2$
+        mapCharset("utf-8", Charset.forName("UTF-8")); //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
This change is not needed, the fallback in getCharset will already return the UTF-8 charset.

Reply:
k